README.md

```diff
@@ -57,7 +57,7 @@ OPTIONS
   --version
 ```
 
-_See code: [src/commands/cdn/index.js](https://github.com/warehouseai/wrhs/blob/v1.1.0/src/commands/cdn/index.js)_
+_See code: [src/commands/cdn/index.js](https://github.com/warehouseai/wrhs/blob/master/src/commands/cdn/index.js)_
```

Comment:
No need to update the readme. It gets auto updated when u generate a new version

Reply:
Hmm that's odd because the version you have is `v7.0.0` but this is pointing to `v1.1.0`. Is that done via some github magic? It could be because the package version and the tags are mismatched 😞